data/countries/de/data.yaml

```
@@ -109,5 +109,60 @@ urls:
     url: https://www.robert-habeck.de/
   - type: WIKIDATA
     url: https://www.wikidata.org/wiki/Q109943
+  - type: INSTAGRAM
+    url: https://www.instagram.com/robert.habeck
 emails:
   - address: robert.habeck@gruene.de
+---
+type: PERSON
+name: Katrin Göring-Eckardt
+family_name: Göring-Eckardt
+given_name: Katrin
+country: DE
+level: DE:BUNDESVERBAND
+urls:
+  - type: WEBSITE
+    url: https://www.goering-eckardt.de/
+  - type: WIKIDATA
+    url: https://www.wikidata.org/wiki/Q48967
+  - type: FACEBOOK
+    url: https://www.facebook.com/GoeringEckardt
+  - type: TWITTER
+    url: https://twitter.com/GoeringEckardt
+  - type: INSTAGRAM
+    url: https://www.instagram.com/goeringeckardt/
+---
+type: PERSON
+name: Toni Hofreiter
```

```suggestion
name: Anton Hofreiter
```

Ich glaube, das ist geläufiger